frontend/www/js/omegaup/components/contest/AddProblem.vue

```diff
@@ -25,11 +25,13 @@
               v-bind:show-footer="false"
               v-model="selectedRevision"
               v-on:runs-diff="onRunsDiff"></omegaup-problem-versions>
-        <div class="form-group">
-          <button class="btn btn-primary add-problem"
-               type="submit">{{addProblemButtonLabel}}</button>
-        </div>
       </form>
+      <div class="form-group">
+        <button class="btn btn-primary add-problem"
+             type="submit"
+             v-on:click="onAddProblem"
```

Comment:
```suggestion
             v-on:click.prevent="onAddProblem"
```

si no, se va a llamar `onSubmit` también.

Reply:
Listo,